launcher/src/components/UI/node-manage/SidebarManage.vue

```diff
@@ -150,6 +150,41 @@ export default {
           break;
       }
     },
+    selectCategoryTitlePrev() {
```

Comment:
this will work but in software development is `KISS` -> `Keep It Small & Simple` the key

you made an almost exact copy of an function so basically you have now lines in your code that have the exact same perpuse
you can pass variables like this: `@click="selectCategoryTitle("right")"` for example
try adjusting the existing function
you can `ADD` -> `COMMIT` -> `PUSH` on this branch and it will update the PR as well